hieradata/clients/azure.ci.jenkins.io.yaml

```diff
@@ -415,6 +448,16 @@ profile::jenkinscontroller::jcasc:
           virtualNetworkResourceGroupName: "prod-jenkins-public-prod"
           subnetName: "ci.j-agents-vm"
           spot: true
+          initScript:
+            - "sed 's/api_key:.*/api_key: ENC[PKCS7,MIIBmQYJKoZIhvcNAQcDoIIBijCCAYYCAQAxggEhMIIBHQIBADAFMAACAQEwDQYJKoZIhvcNAQEBBQAEggEAiiTWRyFEQbPaJ8OHohWOhICF/mxCHioTCk7QhxEXZ+PD6hFmKvhZFjPb5qG52MKHA1hHZtC5NnXp+BSpmoE2TK9DLQstzZxA3xsuoBcLiz5o3J3HqV4Vz9SKNC2gUtCnBFN2IQLbKQvnHeHFzPyltzWIJ4+ZWtH5RymSqDpkxBEw8IeIwX8Ps9Ryhmip3DJrWmtogpAdhfwlTzHbwDJvQkl7GjMzdPAEEISy0rtI2nj+HSFgiG7iCfAVIy+DcEFcgFrEX3xmA+IeurOT3Q1ayt1KJCgtTtO27GcJDOydeFJIIZluwnx4OnkmvZPCYcXh1KGiDvm+8sw3eklMSeswZDBcBgkqhkiG9w0BBwEwHQYJYIZIAWUDBAEqBBCJDjHErg9iT8ltw26SaB8TgDD2TVMoUmvWfiwUEz0hJsTJUsW4cg2iavGCJk2x4+m4BhljtPM5W5dfMJjgGtRe0Nc=]/' /etc/datadog-agent/datadog.yaml.example > /etc/datadog-agent/datadog.yaml"
```

Comment:
Is it normal these 2 lasts api_key are different than the 3 above?

Reply:
if you talk about the encrypted ones ... once decrypted they are all the same for the 7 agents

Reply:
I don't understand why the 3 first are equals while the 2 lasts aren't. Typo?

Reply:
It's because of the encryption: each instance has a different salt so it's less deterministic (but of course, salting remove the possibility to de-duplicate the encrypted data).